apps/server/src/modules/index.ts

```diff
@@ -37,4 +40,13 @@ switch (SERVER_FLAVOR) {
     break;
 }
 
-export { BusinessModules };
+const Providers: Provider[] = [];
+
+if (NODE_ENV === 'production') {
```

Comment:
why production only?

Reply:
It makes test output ugly

Reply:
how about NODE_ENV !== 'test' https://github.com/toeverything/AFFiNE/blob/cb0d9fa22aaf5eb130ebc8ba92b72e23b9a76bcd/apps/server/scripts/run-test.ts#L25